File: backend/portfolio-backend/src/main/java/com/mattysleduc/portfolio/domain/testimonialsubdomain/presentation_layer/TestimonialRequestModel.java
package com.mattysleduc.portfolio.domain.testimonialsubdomain.presentation_layer;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class TestimonialRequestModel {
    private String authorName;
    private String authorTitle;
    private String authorTitleEn;
    private String authorTitleFr;
    private String authorImage;
    private String content;
    private String contentEn;
    private String contentFr;
    private Integer rating;
    private String company;
}

File: frontend/src/features/testimonials/models/TestimonialRequestModel.ts
export interface TestimonialRequestModel {
  authorName: string;
  authorTitle?: string;
  authorTitleEn?: string;
  authorTitleFr?: string;
  authorImage?: string;
  content?: string;
  contentEn: string;
  contentFr: string;
  rating: number;
  company?: string;
  companyEn?: string;
  companyFr?: string;
}
